OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -53,6 +53,10 @@ public class MobileInfo : PausableConditionalTraitInfo, IMoveInfo, IPositionable
 		[Desc("Boolean expression defining the condition under which the regular (non-force) move cursor is disabled.")]
 		public readonly BooleanExpression RequireForceMoveCondition = null;
 
+		[ConsumedConditionReference]
+		[Desc("Boolean expression defining the condition under which this actor cannot be nudged by other actors.")]
+		public readonly BooleanExpression ImmovableCondition = null;
```

Comment:
In some places we use Immobile and others Immovable. It would be good if we could standardize on one or the other.

Reply:
The name was chosen to be consistent with the usage in `BlockedByActor.Immovable`, `CellFlag.HasMovableActor` and `CellCache.Immovable` among others. Those names where deliberately chosen to express a subtle difference in meaning with immobile: These categories also include actors that are quite mobile, but aren't movable by others (such as enemy units).

`Immobile` is actually only used as the name of a trait and nowhere else.